app/src/main/java/com/example/androidtrainingproject/ui/home/HomeScreen.kt

```diff
@@ -162,6 +229,136 @@ fun Home(navController: NavController) {
+                                    )
+
+                                    coroutineScope.launch {
+                                        sheetState.hide()
+                                    }
+                                }
+                            )
+                        }
+                        CategoriesAccordion(
+                            homeViewModel = homeViewModel,
+                            checkedCategories = checkedCategories,
+                            ::checkCategory,
+                            ::unCheckCategory
+                        )
+                        Text(
+                            text = stringResource(id = R.string.price_range),
+                            modifier = Modifier.padding(top = dimensionResource(id = R.dimen.padding_m)),
+                            style = MaterialTheme.typography.titleSmall
+                        )
+                        RangeSlider(
+                            value = priceRange,
+                            onValueChange = { value ->
+                                priceRange = value.start.roundToInt()
+                                    .toFloat()..value.endInclusive.roundToInt().toFloat()
+                            },
+                            valueRange = 0f..200f,
+                            modifier = Modifier.padding(top = dimensionResource(id = R.dimen.padding_m)),
+                            colors = SliderDefaults.colors(
+                                thumbColor = Purple,
+                                activeTrackColor = Purple,
+                                inactiveTrackColor = Purple,
+                                activeTickColor = Purple,
+                                inactiveTickColor = Purple,
+                                disabledThumbColor = Purple,
+                                disabledActiveTrackColor = Purple,
+                                disabledActiveTickColor = Purple,
+                                disabledInactiveTrackColor = Purple,
+                                disabledInactiveTickColor = Purple
+                            )
+                        )
+                        Row(
+                            modifier = Modifier
+                                .padding(top = dimensionResource(id = R.dimen.padding_sm))
+                                .fillMaxWidth(),
+                            horizontalArrangement = Arrangement.SpaceBetween
+                        ) {
+                            Text(
+                                text = "$${priceRange.start}",
+                                style = MaterialTheme.typography.titleSmall
+                            )
+                            Text(
+                                text = "$${priceRange.endInclusive}",
+                                style = MaterialTheme.typography.titleSmall
+                            )
+                        }
+                        Text(
```

Comment:
Extract this `Text` and the `LazyRow` below to a separate composable, exposing a modifier parameter so you can pass the top padding of the Text from outside (as outer paddings shouldn't be defined inside of an inner composable).